app/models/open_humans_profile.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class OpenHumansProfile < ActiveRecord::Base
```

Comment:
For that we'd need Rails5, first... 🐩

Reply:
What exactly do you mean here? Follows the same convention as all other models?

Reply:
Rubocop wants the models to subclass a comon `ApplicationRecord` class. I tried doing this in another project using Rails 4 and it didn't work. But, thinking about it, this may have been due to some gem we are using. Long story short, just ignore it for now. We can do this separately.

Reply:
Ok!